jenkins-scripts/docker/lib/docker_generate_dockerfile.bash

```diff
@@ -33,24 +33,32 @@ export APT_PARAMS=
 
 GZDEV_DIR=${WORKSPACE}/gzdev
 GZDEV_BRANCH=${GZDEV_BRANCH:-master}
+if [[ "$ghprbSourceBranch" =~ matching_branch\/ ]]; then
```

Comment:
Using somehow the branch name to include some internal commands is not a bad idea. Could we somehow introduce the notion of "this is for Jenkins" or "this is a command for infra" in the name? Something like branch name: `*/ci_matching_branch/*` or similar?

Reply:
sure; I'll go for `ci_matching_branch/`, so it doesn't have to have a preceding `/`

Reply:
done in https://github.com/ignition-tooling/release-tools/pull/572/commits/2c6faf12af27a5caf69e208d9eceb3dfe1382e5b